syncer/main.py

```diff
@@ -396,6 +480,9 @@ def update_remote_shutter_state(self, ip, remote_id, state, reverse, is_shutter_
                 remote_state = ""
```

Comment:
Why no remote state in this case?

Reply:
The other options are "up" and "down" which means they are currently not doing anything. To get into the up/down state we have to let it go up/down and wait till its individual shutter timer has stopped. We cannot trigger the transition of going_up to up by a button press, so that's why we won't do anything here.

Also, I did remote_state = "" just so pycharm wouldn't give me any errors about the possibility that remote_state isn't initiated before using.